module/VuFind/src/VuFind/Search/Solr/Options.php

```diff
@@ -40,16 +40,6 @@ class Options extends \VuFind\Search\Base\Options
 {
     use \VuFind\Search\Options\ViewOptionsTrait;
 
-    /**
-     * Available sort options for facets
-     *
-     * @var array
-     */
-    protected $facetSortOptions = [
```

Comment:
Rather than deleting this, should you just reformat it so it's the `*` settings, so if nothing is configured in facets.ini, the existing default behavior continues to work?

Reply:
Sorry, I thought that we could assume that `/config/vufind/facets.ini` is up-to-date. Reverted back to `Options`.

Reply:
Thanks! Your assumption was probably safe in most cases, but I like to be careful just in case people upgrade in non-standard ways.